pkg/controller/configmap/configmap_controller.go

```diff
@@ -139,8 +165,10 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 			operatortypes.OperatorNamespace))
 		watchedNamespace = operatortypes.OperatorNamespace
 	}
-	log.Info(fmt.Sprintf("configmap controller watching events in namespace %s", watchedNamespace))
+	
 	if request.Namespace == watchedNamespace {
+		log.Info(fmt.Sprintf("Received configmap controller watching events in namespace: %s", watchedNamespace))
```

Comment:
I don't think we need this log as we have accurate log in each `if` block.

Reply:
I'm ok for  this one.